packages/zwave-js/src/lib/commandclass/SceneControllerConfigurationCC.ts

```diff
@@ -0,0 +1,490 @@
+		const node = this.getNode()!;
+		const endpoint = this.getEndpoint()!;
+		const api = endpoint.commandClasses[
+			"Scene Controller Configuration"
+		].withOptions({
+			priority: MessagePriority.NodeQuery,
+		});
+
+		this.driver.controllerLog.logNode(node.id, {
+			message: `${this.constructor.name}: doing a ${
+				complete ? "complete" : "partial"
+			} interview...`,
+			direction: "none",
+		});
+
+		const groupCount = this.getGroupCountCached();
+		if (groupCount === 0) {
+			this.driver.controllerLog.logNode(node.id, {
+				endpoint: this.endpointIndex,
+				message: `soft failing interview (missing association group count)`,
+				direction: "none",
+			});
+
+			this.interviewComplete = false;
+			return;
+		}
+
+		this.driver.controllerLog.logNode(node.id, {
+			endpoint: this.endpointIndex,
+			message: `supports ${groupCount} association groups`,
+			direction: "none",
+		});
+
+		// Always query scene and dimmer duration for each association group
+		// skipping group #1, which is reserved for Lifeline
+		for (let groupId = 2; groupId <= groupCount; groupId++) {
+			this.driver.controllerLog.logNode(node.id, {
+				endpoint: this.endpointIndex,
+				message: `querying association group #${groupId}...`,
+				direction: "outbound",
+			});
+			const group = await api.get(groupId);
+			if (group != undefined) {
+				const logMessage = `received information for association group #${groupId}:
+scene Id: ${group.sceneId}
+dimming duration: ${group.dimmingDuration.toString()}`;
+				this.driver.controllerLog.logNode(node.id, {
+					endpoint: this.endpointIndex,
+					message: logMessage,
+					direction: "inbound",
+				});
+			}
+		}
+
+		// Remember that the interview is complete
+		this.interviewComplete = true;
+	}
+
+	/**
+	 * Returns the number of association groups reported by the node.
+	 * This only works AFTER the node has been interviewed by this CC
+	 * or the AssociationCC.
+	 */
+	public getGroupCountCached(): number {
+		return this.getValueDB().getValue(getGroupCountValueId()) ?? 0;
+	}
+}
+
+interface SceneControllerConfigurationCCSetOptions extends CCCommandOptions {
+	groupId: number;
+	sceneId: number;
+	dimmingDuration: Duration;
+}
+
+@CCCommand(SceneControllerConfigurationCommand.Set)
+export class SceneControllerConfigurationCCSet extends SceneControllerConfigurationCC {
+	public constructor(
+		driver: Driver,
+		options:
+			| CommandClassDeserializationOptions
+			| SceneControllerConfigurationCCSetOptions,
+	) {
+		super(driver, options);
+		if (gotDeserializationOptions(options)) {
+			// TODO: Deserialize payload
+			throw new ZWaveError(
+				`${this.constructor.name}: deserialization not implemented`,
+				ZWaveErrorCodes.Deserialization_NotImplemented,
+			);
+		} else {
+			const groupCount = this.getGroupCountCached();
+			this.groupId = options.groupId;
+			this.sceneId = options.sceneId;
+			this.dimmingDuration = options.dimmingDuration;
+
+			if (this.groupId < 2 || this.groupId > groupCount) {
```

Comment:
From the specs:
> A sending node SHOULD NOT specify Group ID 1

I think since it is not forbidden, we should allow it, because one might want to configure the command that is sent to the lifeline